test/bigtest/tests/service-point-create-test.js

```diff
@@ -10,13 +10,7 @@ describe('ServicePointCreate', () => {
 
   beforeEach(function () {
     servicePoint = this.server.create('servicePoint');
-  });
-
-  beforeEach(function () {
     this.visit(`/settings/tenant-settings/servicePoints/${servicePoint.id}?layer=edit`);
```

Comment:
Can this be removed at all?

Reply:
Yes. Both [`server.create`](https://github.com/miragejs/miragejs/blob/master/lib/server.js#L861) and [`this.visit`](https://github.com/bigtestjs/react/blob/master/src/visit.js#L20) are synchronous so we can combine the `beforeEach`.

Reply:
I mean that there two consecutive this.visit calls. As I understand only the last one is needed.

Reply:
Oh, I mean that was there before and doesn't relate to the mirage refactor. We'll reevaluate the tests on a different PR.

Reply:
Sounds goog.